storage/range_raftstorage.go

```diff
@@ -331,6 +331,22 @@ func (r *Range) Append(entries []raftpb.Entry) error {
 	return nil
 }
 
+// updateRangeInfo is called whenever a range is updated by ApplySnapshot
+// to setup the fields which are uninitialized or need updating.
+func (r *Range) updateRangeInfo() error {
+	// Update the max bytes of the range after snapshot applying.
+	// If no updating of zoneConfigs for this key range, then
+	// maxBytes of this range will not be updated.
+	zone, err := lookupZoneConfig(r.rm.Gossip(), r)
+	if err != nil {
+		return util.Errorf("couldn't find a zone config for range %v", r)
```

Comment:
- No need to use `%v` here. `Range` implements `Stringer`.
- We should include the original error
How about: `return util.Errorf("failed to look up zone config for Range %s: %s", r, err)`?